scripts/podman-volume-backup.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Get a list of all volumes
+volumes=$(podman volume ls -q)
+#get timestamp for unique filename
+timestamp=$(date +"%Y%m%d%H%M%S")
+
+# Temp directory for tar vols
+mkdir temp_podman_backup_${timestamp}
```

Comment:
This is an improvement, thanks.

In case your interested (I'm not going to block this PR over it): The pattern I normally follow for temp. dirs goes something like:

```bash
DEBUG=${DEBUG:-0}
TEMPDIR=$(mktemp -p '' -d temp_podman_backup_XXXX)
trap "if ! ((DEBUG)); then rm -rf $TEMPDIR; fi" EXIT

...code...
```

This guarantees the temp. dir will always have a unique name.  Then, it automatically removes the directory when the script exits, **unless** the script is called with `DEBUG=1` set.

Again, I won't hold you to it for this PR, it could easily be done as a future improvement.